core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/function/aggregate/var/VarianceAggregateFactory.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eclipse RDF4J contributors.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *******************************************************************************/
+package org.eclipse.rdf4j.query.algebra.evaluation.function.aggregate.var;
+
+import java.util.function.Function;
+
+import org.eclipse.rdf4j.model.Value;
+import org.eclipse.rdf4j.query.BindingSet;
+import org.eclipse.rdf4j.query.algebra.evaluation.function.aggregate.StatisticalAggregateFunction;
+import org.eclipse.rdf4j.query.parser.sparql.aggregate.AggregateCollector;
+import org.eclipse.rdf4j.query.parser.sparql.aggregate.AggregateFunction;
+import org.eclipse.rdf4j.query.parser.sparql.aggregate.AggregateFunctionFactory;
+
+/**
+ * {@link AggregateFunctionFactory} implementation that provides {@link AggregateFunction} used for processing
+ * population variance
+ *
+ * @author Tomas Kovachev t.kovachev1996@gmail.com
+ */
+public class VarianceAggregateFactory implements AggregateFunctionFactory {
+
+	@Override
+	public String getIri() {
+		return "http://rdf4j.org/aggregate#variance";
```

Comment:
I'm wondering if maybe the most common use for `variance` is sample based and that we should have a separate population based variance instead.

Python uses sample based as default: https://appdividend.com/2022/03/10/python-variance/
Apache commons math also uses sample based as default: https://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math4/stat/descriptive/SummaryStatistics.html#getVariance--